src/displayBackend/common/drm_edid.h

```diff
@@ -314,6 +316,8 @@ struct edid {
 	/* Standard timings 1-8*/
 	struct std_timing standard_timings[8];
 	/* Detailing timings 1-4 */
+	/* First descriptor is used for detailed timing
```

Comment:
You are trying to comment your specific use-case in a generic structure which is not entirely correct, IMO
Please remove

Reply:
Ok, will move it to the other place